doc/connectivity/bluetooth/api/mesh/brg_cfg.rst

```diff
@@ -113,13 +113,16 @@ protection to ensure network security. Key considerations to take into account a
 Relay buffer considerations
 ===========================
 
-When a message is relayed between subnets by a Subnet Bridge, it is allocated from the relay buffer.
-To ensure that messages can be retransmitted to all subnetworks,
-the :kconfig:option:`CONFIG_BT_MESH_RELAY_BUF_COUNT` option should be increased accordingly.
-
-However, if the :kconfig:option:`CONFIG_BT_MESH_RELAY` feature is disabled, the messages will be
-allocated from the advertising buffer instead. In this case, increase the
-:kconfig:option:`CONFIG_BT_MESH_ADV_BUF_COUNT` option to allow for sufficient buffer space.
+When a message is relayed between subnets by a Subnet Bridge, it is allocated from the relay buffer
+pool and transmitted using the relay advertising sets. These will be used even if the relay feature
+:kconfig:option:`CONFIG_BT_MESH_RELAY` is disabled. If enabled, the relay buffers and advertising
+sets will use the parameters set by :kconfig:option:`CONFIG_BT_MESH_RELAY_ADV_SETS` and
+:kconfig:option:`CONFIG_BT_MESH_RELAY_BUF_COUNT`. If relaying is disabled, the parameters set by
+:kconfig:option:`CONFIG_BT_MESH_BRG_ADV_SETS` and :kconfig:option:`CONFIG_BT_MESH_BRG_BUF_COUNT`
+will be used.
+
+To ensure that messages can be retransmitted to all subnetworks, the buffer count option should be
+increased accordingly.
```

Comment:
The value of the RELAY_BUF_COUNT/BRG_BUF_COUNT will impact the relay/bridge latency directly. The default value is choosen to optimize good latency with lesser packet loss. Therefore, I will strongly suggest to remove this recommendation to avoid adding confusion.